sycl/source/detail/scheduler/commands.cpp

```diff
@@ -1299,10 +1311,6 @@ const ContextImplPtr &MemCpyCommand::getWorkerContext() const {
   return getWorkerQueue()->getContextImplPtr();
 }
 
-const QueueImplPtr &MemCpyCommand::getWorkerQueue() const {
-  return MQueue->is_host() ? MSrcQueue : MQueue;
-}
-
```

Comment:
What's reasoning for removing of this method?

Reply:
MWorkerQueue is assigned in Command constructor with default value and in specific command accessors constructor reassigned to another value (if it differs from default).
that means that we can just implement method getWorkerQueue that just returns WorkerQueue - it is done and located in Command class. getWorkerQueue overloading in accessors removed.